openapi/aayu.mftg.as2/Dependencies.toml

```diff
@@ -54,7 +54,7 @@ dependencies = [
 [[package]]
 org = "ballerina"
 name = "http"
-version = "2.2.1"
+version = "2.2.2"
```

Comment:
Need to delete and update the Dependencies.toml
Correct http verison will be `2.4.0`

Reply:
As we discussed @LakshanSS ,

We cannot build these open API connectors with HTTP 2.4.0 because of this issue.
e.g:
```
ERROR [client.bal:(66:53,66:65)] incompatible types: expected 'ballerina/http:2.4.0:ClientConfiguration', found 'ballerinax/bitbucket:1.3.0:ClientConfig'
```
So we will go with existing HTTP versions for this release of 2201.2.1. 

(We can update dependencies after regenerating the connectors from the open-API tool with config changes)

Reply:
+1